### DISCUSSION TITLE
Configurable HTTP code for concurrency limits

### DISCUSSION THREAD

**Discussion by yalwan-sage** [Kludex/uvicorn#2301] [Ideas] [answered]
I was looking at https://github.com/encode/uvicorn/blame/a2219eb2ed2bbda4143a0fb18c4b0578881b1ae8/uvicorn/protocols/http/flow_control.py

because I'd rather have a 429 than 503 for concurrency limits reached; I noticed that there's a magic number in the file which isn't used (`CLOSE_HEADER`), but also that the 503 is hardcoded.

What do the maintainers feel about making this configurable?

**Comment by Kludex** [comment, marked as answer]
`CLOSE_HEADER` is being used.

503 seems the right status code for it.

I'd prefer to not allow it to be configurable.
